<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialUnion</class>
 <widget class="QMainWindow" name="SerialUnion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSCAN-V1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rc.qrc">
    <normaloff>:/imageFile/font.ico</normaloff>:/imageFile/font.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="12,1" columnstretch="24,1,1">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="widgetDisplay"/>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="widgetSettings"/>
      </item>
      <item row="0" column="2" rowspan="2">
       <layout class="QHBoxLayout" name="widgettest"/>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="widgetSend"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="mbSaveRecData"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="mbSaveRecData">
   <property name="text">
    <string>保存接收</string>
   </property>
  </action>
  <action name="mbMoreSettings">
   <property name="text">
    <string>更多设置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../rc.qrc"/>
 </resources>
 <connections/>
</ui>
